transform/relations/relations.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from relations.process_data import (
+    process_combined_directories,
+    process_organization_project_relations,
+    process_result_relations,
+)
+from schemas.properties.relations import *
+
+
+def process_relations():
+    """"""
```

Comment:
empty doc string

Reply:
Yes, I forgot 😅. Thank you for noticing.